### ISSUE TITLE
The rear camera image is flipped 180 degrees

### ISSUE THREAD

**Issue by mymirror** [Blancduman/mediasoup-client-flutter#73]
When we used the rear camera to video call, the image on the far side flipped 180 degrees

**Comment by Blancduman** [comment]
`rtpCapabilities.headerExtensions.removeWhere((he) => he.uri == 'urn:3gpp:video-orientation');`

**Comment by mymirror** [comment]
> `rtpCapabilities.headerExtensions.removeWhere((he) => he.uri == 'urn:3gpp:video-orientation');`

Yes, I did this step, the camera switched to the rear camera, but the remote picture flipped 180 degrees。 
The voice you hear on the iphone is too small, is it turned on by default in handset mode rather than speaker mode? If so, how do I turn on speaker mode
